docs/scos/user/features/202304.0/push-notification-feature-overview.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Push Notification feature overview
+description: The Push Notifications feature lets you receive push notifications
+last_updated: Feb 2, 2023
+template: concept-topic-template
+---
+
+The *Push Notification* feature lets users subscribe to the web push notifications, which are sent from the server to all registered subscriptions.
+
+{% info_block infoBox "Note" %}
+
+The feature can be integrated into the non-Srpyker customer application only. Spryker features do not support push notifications.
+
+{% endinfo_block %}
+
+## Subscribing to notifications
+
+There are two tasks that enable the feature. The first lets the user subscribe to the web push notifications. For this, the site page needs a service worker with which the user is registered on the server and received messages are processed. The service worker is downloaded to the user’s platform in the background, which lets it be executed outside of the site page's context.
```

Comment:
What's the second task?

Reply:
That was written based on the primary source. However, even the primary source does not have the second task mentioned. I'll get rid of that part.